src/js/navIcons.js

```diff
@@ -0,0 +1,115 @@
+/*
+
+Copyright 2015 OCAD University
+
+Licensed under the Educational Community License (ECL), Version 2.0 or the New
+BSD license. You may not use this file except in compliance with one these
+Licenses.
+
+You may obtain a copy of the ECL 2.0 License and BSD License at
+https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
+*/
+
+(function ($, fluid) {
+
+    "use strict";
+
+    fluid.registerNamespace("gpii.firstDiscovery");
+
+    /*
+     * The nav icon
+     */
+    fluid.defaults("gpii.firstDiscovery.icon", {
+        gradeNames: ["fluid.viewComponent", "autoInit"],
+        position: null,  // must be supplied by integrators
+        selectors: {
+            activeIndicator: ".gpiic-fd-activeIndicator",
+            doneIndicator: ".gpiic-fd-doneIndicator"
+        },
+        styles: {
+            active: "gpii-fd-active",
+            show: "gpii-fd-show"
+        },
+        modelListeners: {
+            currentPanelNum: {
```

Comment:
i think this component should have a model like `isActive` and the logic for determining that can be moved into the integration of it that you have in the navIcons.